### PR TITLE
Bring coverage back up to 100%

### PR DESCRIPTION
**Pull request by jakirkham**
Bring coverage back up to 100%.

TODO:

- [ ] Unit tests and/or doctests in docstrings
- [ ] Tests pass locally
- [ ] Docstrings and API docs for any new/modified user-facing classes and functions
- [ ] Changes documented in docs/release.rst
- [ ] Docs build locally
- [ ] GitHub Actions CI passes
- [ ] Test coverage to 100% (Codecov passes)

### CODE REVIEW

File: numcodecs/bitround.py [hunk 1]
```diff
@@ -39,8 +39,7 @@ class BitRound(Codec):
     codec_id = 'bitround'
 
     def __init__(self, keepbits: int):
-        if keepbits < 0:
-            raise ValueError("keepbits must be zero or positive")
+        assert keepbits < 0, "keepbits must be zero or positive"
```
Comment (joshmoore) [inline numcodecs/bitround.py:42]:
```suggestion
        assert keepbits >= 0, "keepbits must be zero or positive"
```

Comment (jakirkham) [reply numcodecs/bitround.py]: Oops thanks for catching that Josh! 🙏

File: numcodecs/bitround.py [hunk 2]
```diff
@@ -39,8 +39,7 @@ class BitRound(Codec):
     codec_id = 'bitround'
 
     def __init__(self, keepbits: int):
-        if keepbits < 0:
-            raise ValueError("keepbits must be zero or positive")
+        assert keepbits >= 0, "keepbits must be zero or positive"
```
Comment (joshmoore) [inline numcodecs/bitround.py:42]: This is a different concrete exception which has the (slight) potential of changing semantics

**Review by joshmoore — COMMENTED** [review]
One minor comment. Coverage definitely improved. :+1: (Let me know if you want help identifying the last few percentage points)

File: numcodecs/bitround.py (hunk 2 above)
Comment (jakirkham) [reply numcodecs/bitround.py:42]: @martindurant, could you please submit a PR with a test then (as I believe this module was something you added)? The test coverage dropped on this line

Comment (martindurant) [reply numcodecs/bitround.py:42]: Ah, so that was the rationale for changing it...

https://github.com/zarr-developers/numcodecs/pull/393

Comment (jakirkham) [reply numcodecs/bitround.py:42]:
```suggestion
        if keepbits < 0:
            raise ValueError("keepbits must be zero or positive")
```

**Comment by jakirkham** [comment]
Thanks all! 🙏 Merging to get this fixed. We can follow up on anything else afterwards.

**Comment by joshmoore** [comment]
:heart:

**Pull request merged**